css/types/calc.json

```diff
@@ -280,6 +280,56 @@
               "deprecated": false
             }
           }
+        },
+        "media_query_values": {
+          "__compat": {
+            "description": "Removal of <a href='https://developer.mozilla.org/docs/Web/CSS/Media_Queries'>Media query</a> value support",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "57",
+                "version_removed": "59",
+                "notes": "In Firefox 57 desktop, <code>calc()</code> was temporarily disabled in media query values during the changeover to Stylo. In Firefox 59, support has been added in again."
```

Comment:
"desktop" is useless as it is the entry for FIrefox desktop. " during the changeover to Stylo" is not really exact, it wasn't disabled during the changeover, but because, but I will let this phrase out anyway as web devs don't need this info.